### PR TITLE
GH action target develop & master branch

### PR DESCRIPTION
**Pull request by royki**
Migrate from Travis CI to GH Actions

### CODE REVIEW

File: .github/workflows/deploy.yml [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+      - name: Test
+        run: npm run test
+
+      - name: Deploy on develop branch
+        if: github.ref == 'refs/heads/develop'
+        run: npx semantic-release
+
+      - name: Deploy on master branch
+        if: github.ref == 'refs/heads/master'
+        run: npx semantic-release
```
Comment (jrhender) [inline .github/workflows/deploy.yml:55]: @royki Could we add the same commit back into develop that we have in the iam-client-lib build as well here please?

File: .github/workflows/deploy.yml [hunk 2]
```diff
@@ -0,0 +1,55 @@
+name: Deploy
+
+on:
+  push:
+    branches: [ 'develop', 'master' ]
+  pull_request:
+    branches: [ 'develop' ]
```
Comment (jrhender) [inline .github/workflows/deploy.yml:7]: Why just develop and not master as well here?

**Review by jrhender — COMMENTED** [review]
Just a couple comments. Also please remove .travis.yml file.

Would be good to get @knzeng-e 's review as well as he is responsible for passport-did-auth now

File: .github/workflows/deploy.yml (hunk 2 above)
Comment (royki) [reply .github/workflows/deploy.yml:7]: Ok, that's done.

File: .github/workflows/deploy.yml (hunk 1 above)
Comment (royki) [reply .github/workflows/deploy.yml:55]: You mean this one - ?
```
   - name: Deploy on master branch
        if: github.ref == 'refs/heads/master'
        run: |
          npx semantic-release
          git checkout develop
          git merge origin/master
          git push origin develop
```

**Review by jrhender — APPROVED** [review]
Looks good

**Review by knzeng-e — APPROVED** [review]
Sounds good, thanks a lot @royki  !

**Pull request merged**